src/map/GMap.js

```diff
@@ -258,6 +268,30 @@
         this.updateCircles();
         this.updatePins();
     };
+    
+    GMap.prototype.showStreetView = function () {
+        var map;
+        if(this.streetViewTargetSelector()){
+            map = new google.maps.Map(document.querySelector(this.streetViewTargetSelector()),{zoom: 5,mapTypeId: google.maps.MapTypeId.ROADMAP});
```

Comment:
Yes, exactly. It will render the Street View in another html element if **streetViewTargetSelector** is specified. I'm now realizing that I haven't actually tested this feature, but it *should* work. I'll test it and remove it from this PR if it doesn't work perfectly.